### ISSUE TITLE
How many epoches before the LSTM model overfit?

### ISSUE THREAD

**Issue by starrywintersky** [LukeTheHecker/esinet#6]
I have just copy the quick_start file to a python file, and run it with 1000 samples and 150 epoches, with the CosineSimilarity loss and 'mse'/'mae' evaluations. However, I got an increase of MSE and MAE both for training and evaluation set after only 7 epoches. 
I wondor if it is normal, or I have set some hyperparameters wrong ?

**Comment by LukeTheHecker** [comment]
Hi, it is possible for MAE and MSE to increase while CosineSimilarity loss decreases (i.e., become more negative). This is because CosineSimilarity does only compare similarity between true and predicted sources without offsets. See for example these two vectors, which could be low resolution source vectors:

y_true = [3, 4, 5, 3]
y_pred = [6, 8, 10. 6]

While, clearly, MAE and MSE are not zero, the correlation between the two vectors is perfect, Hope this demonstrates that MAE and MSE are not important in that sense, since y_pred would only need to be multiplied by a scalar (0.5) to get to the true source.

Hope that helped, feel free to come back the issue if you have doubts.

Best,
Lukas
